com.microsoft.mrtk.input/Controllers/ArticulatedHandController.cs

```diff
@@ -40,7 +40,7 @@ public class ArticulatedHandController : ActionBasedController
         #region Properties
         
         [Obsolete("Deprecated, please use XRSubsystemHelpers.HandsAggregator instead.")]
-        protected HandsAggregatorSubsystem HandsAggregator => XRSubsystemHelpers.HandsAggregator;
+        protected HandsAggregatorSubsystem HandsAggregator => (HandsAggregatorSubsystem)XRSubsystemHelpers.HandsAggregator;
```

Comment:
Can this be changed to IHandsAggregatorSubsystem ?    XRSubsystemHelpers may hold something that is not a HandsAggregatorSubsystem.

If it can't be changed, use the "as" operator.